aries-site/src/pages/components/checkbox.mdx

```diff
@@ -34,6 +34,10 @@ There are three states which CheckBox provides:
 
 Checked and unchecked states refer to when an individual CheckBox is checked or when all options from a group are checked.
 
+### CheckBox outside of FormField
+The CheckBox has a built in `pad` to allow space when wrapped in a FormField. In order to have a stand alone CheckBox align with other FormFields in a Form users can explicatly 
+set the `pad: {{ vertical: 'xsmall' }}` in order to remove the horizontal pad. 
```

Comment:
I thought we don't want any vertical pad outside of FormField, and that we're going to handle the hover treatment differently when outside of FormField

Reply:
so should we say pad="none"? or say that pad can be controlled via `pad` prop and value set to accomodate the layout, but that mostly commonly it would be "none"?

Reply:
let me put this in draft, for now, sorry forgot about hover treatment I'll update once I chat with designers